DemoApp/LoginViewController.swift

```diff
@@ -101,12 +112,24 @@ extension LoginViewController: UITableViewDelegate, UITableViewDataSource {
     }
     
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        guard indexPath.row != builtInUsers.count else {
+        if indexPath.row == builtInUsers.count {
+            // Anonymous user
+            didRequestChatPresentation(.anonymous)
+        } else if indexPath.row == builtInUsers.count + 1 {
+            // Guest user
+            presentAlert(title: "Input a user id", message: nil, textFieldPlaceholder: "guest") { [weak self] userId in
+                if let userId = userId, !userId.isEmpty {
+                    self?.didRequestChatPresentation(.guest(userId))
+                } else {
+                    self?.didRequestChatPresentation(.guest("guest"))
+                }
+            }
+        } else if indexPath.row == builtInUsers.count + 2 {
```

Comment:
As mentioned before, the enums will also make this easier. Right now it is very fragile, if add new options, it will be required to change all these hardcoded values.